src/ReactiveDomain.Foundation/StreamStore/JsonMessageSerializerSettings.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Reflection;
+
+// ReSharper disable once CheckNamespace
+namespace ReactiveDomain.Foundation {
+    public class JsonMessageSerializerSettings
+    {
+        public bool ThrowOnTypeNotFound;
+        public Assembly AssemblyOverride;
+        public bool FullyQualify;
```

Comment:
Can we put the same XML comments on the public properties, and one on the class itself?